### PR TITLE
Make Stash chart namepaced

### PR DESCRIPTION
**Pull request by tamalsaha**

### CODE REVIEW

**Comment by tamalsaha** [comment]
@unguiculus chart is namespaced

File: stable/stash/templates/deployment.yaml [hunk 1]
```diff
@@ -1,29 +1,36 @@
...
+      serviceAccountName: {{ if .Values.rbac.create }}{{ template "stash.fullname" . }}{{ else }}"{{ .Values.rbac.serviceAccountName }}"{{ end }}
       containers:
       - args:
         - run
         - --v=3
-        image: '{{ .Values.operator.image }}:{{ .Values.operator.tag }}'
-        imagePullPolicy: '{{ .Values.operator.pullPolicy }}'
+        - --rbac={{ .Values.rbac.create }}
+        image: {{ .Values.operator.image }}:{{ .Values.operator.tag }}
+        imagePullPolicy: {{ default "" .Values.imagePullPolicy | quote }}
```
Comment (unguiculus) [inline stable/stash/templates/deployment.yaml]: Remove the `default` function. It's unnecessary.

File: stable/stash/templates/deployment.yaml [hunk 2]
```diff
@@ -34,7 +41,11 @@ spec:
         - -web.listen-address=:56789
         - -persistence.file=/var/pv/pushgateway.dat
         image: '{{ .Values.pushgateway.image }}:{{ .Values.pushgateway.tag }}'
-        imagePullPolicy: '{{ .Values.pushgateway.pullPolicy }}'
+        imagePullPolicy: {{ default "" .Values.imagePullPolicy | quote }}
```
Comment (unguiculus) [inline stable/stash/templates/deployment.yaml]: Remove the `default` function. It's unnecessary.

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by tamalsaha** [comment]
Addressed comments. PTAL.

**Review by unguiculus — APPROVED** [review]

**Pull request merged**